data/battle_scripts_1.s

```diff
@@ -8033,18 +8034,59 @@ BattleScript_SupremeOverlordActivates::
 
 BattleScript_CostarActivates::
 	pause B_WAIT_TIME_SHORT
-	call BattleScript_AbilityPopUp
+	call BattleScript_AbilityPopUpScripting
 	printstring STRINGID_PKMNCOPIEDSTATCHANGES
```

Comment:
This string uses `gBattlerAttacker` and `gBattlerTarget` and likely doesn't use the correct mon's name since `gBattlerAttacker` isn't changed anymore. The string is also used by Psych Up.

Reply:
Right. I always forget about strings. Since it is used for Psych up I reverted the change. Looking at Coaster it might not cause any issues because it happens on switch in.